overlays/bootstrap.nix

```diff
@@ -109,6 +109,7 @@ in {
                 ++ self.lib.optional (version == "8.6.5") ./patches/ghc/ghc-8.6.5-atomic-arm-arch.patch
                 ++ self.lib.optional (version == "8.8.1") ./patches/ghc/ghc-8.8.1-reinstallable-lib-ghc.patch
                 ++ self.lib.optional (version == "8.8.2") ./patches/ghc/ghc-8.8.2-reinstallable-lib-ghc.patch
+                ++ self.lib.optional (version == "8.8.3") ./patches/ghc/ghc-8.8.2-reinstallable-lib-ghc.patch
```

Comment:
We've got three versions with this now, why not put it in a `fromUntil`?

Reply:
Actually there's only two 8.8.1 has its own (slightly different patch).  I think the fact that was not clear is an even better argument for a `fromUntil` though (so we have one line per patch).

Reply:
Oops, yes indeed!